docs/support_matrix.rst

```diff
@@ -35,5 +35,5 @@ Supported NVIDIA hardware, CUDA, OS, and CUDA driver
   | - Turing                         |               | see `enhanced CUDA compatibility guide <https://docs.nvidia.com/deploy/cuda-compatibility/index.html#enhanced-compat-minor-releases>`__ | - RHEL 8                                                                                                       |                         |                     |                    |                                                               |
   | - Volta                          |               |                                                                                                                                         | - and other `PEP599 - The manylinux2014 Platform Tag <https://www.python.org/dev/peps/pep-0599/>`__ compatible |                         |                     |                    |                                                               |
   +----------------------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+-------------------------+---------------------+--------------------+---------------------------------------------------------------+
-  | - Xavier                         | Not Available | 11.8                                                                                                                                    | Jetpack 5.0.2                                                                                                  | SM 5.3 and later        | Jetpack 5.0.2       | Jetpack 5.0.2      | - :ref:`Python wheel can be build from source <jetson build>` |
+  | - Tegra                          | Not Available | 12.2                                                                                                                                    | Jetpack 5.0.2                                                                                                  | SM 5.3 and later        | Jetpack 5.0.2       | Jetpack 5.0.2      | - :ref:`Python wheel can be build from source <jetson build>` |
```

Comment:
In the cmake file you're updating the minimal SM to 72, is that because of the CI limitations or that is something that should be reflected here too?

Reply:
Reverted